Sources/MediaViewControllers/MediaViewController.swift

```diff
@@ -37,6 +38,8 @@ class VLCMediaViewController: VLCPagingViewController<VLCLabelCell> {
             newCell?.iconLabel.textColor = PresentationTheme.current.colors.orangeUI
         }
         super.viewDidLoad()
+        viewControllers.forEach { ($0 as? VLCMediaCategoryViewController)?.delegate = self
```

Comment:
Nit-pick: `($0 as? VLCMediaCategoryViewController)?.delegate = self` should probably have a return line before.